FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.module.css

```diff
@@ -41,7 +41,8 @@
   font-size: 14px;
   font-style: normal;
   font-weight: 400;
-  line-height: 20px; /* 142.857% */
+  line-height: 20px;
+  /* 142.857% */
```

Comment:
What does this comment mean? Do we need it?

Reply:
Comments removed